### PR TITLE
created mock-interview page

### PR DESCRIPTION
**Pull request by Dhanush-2605**

### CODE REVIEW

File: src/common/components/ViewCards.jsx [hunk 1]
```diff
@@ -35,3 +42,50 @@ export const ViewCards = ({
...
+        {data.length === 0 ? (
+          <LoadingAnimation className="text-black !w-10 !h-10" />
+        ) : (
+          <div className="w-full flex flex-wrap justify-center gap-5 py-20 ">
+            {children}
+          </div>
+        )}
+      </>
+    );
+  };
```
Comment (SarathAdhi) [inline src/common/components/ViewCards.jsx]: use `ViewCards` itself. The component is already created, why to create a new card again?

File: src/pages/interviews/mock/index.jsx [hunk 2]
```diff
@@ -1,23 +1,85 @@
...
+     <Pagination
         page={parseInt(page)}
         Component={Component}
         length={pageLength}
       />
-    </PageLayout>
+     </PageLayout> */}
+    </>
   );
 }
```
Comment (SarathAdhi) [inline src/pages/interviews/mock/index.jsx]: Keep the `PageLayout` and remove all the unnecessary comments and no need for the `Component`, just copy paste it inside the main component itself..

File: src/pages/interviews/mock/index.jsx [hunk 3]
```diff
@@ -1,23 +1,85 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { Pagination } from "../../../common/components/Pagination";
 import PageLayout from "../../../common/layout/PageLayout";
-
+// import { H4, P } from "../../common/components/elements/Text";
```
Comment (SarathAdhi) [inline src/pages/interviews/mock/index.jsx]: remove the comments

**Review by SarathAdhi — CHANGES_REQUESTED** [review]

File: src/common/components/ViewCards.jsx (hunk 1 above)
Comment (Dhanush-2605) [reply src/common/components/ViewCards.jsx]: yeah, but the design, you gave card component for the mock interview page is different from other
image
image

Comment (SarathAdhi) [reply src/common/components/ViewCards.jsx]: Just add a check for description. If the description is there, then display itt

**Pull request merged**